### DISCUSSION TITLE
Fix fluids physics

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#806] [Help]
Hi. I'm trying to fix water and lava (fluids) physics. I mean, I want to implement `level` values for this fluids. I tried to use `BlockHandler`, listen to `PlayerBlockPlaceEvent` and tick water, but nothing is worked. How can I listen to water/lava placement?

**Comment by TheMode** [comment]
How are you placing the liquid? Be aware that buckets are not implemented.

**Reply by ghost** [reply]
I place via buckets...

**Reply by ghost** [reply]
Okay, I'll try to implement buckets first, then
